tools/loader/Loader.cpp

```diff
@@ -406,6 +406,9 @@ void Loader::compile(PlaceholderBindings &bindings) {
     // Emit IR for the graph and compile it.
     auto error = hostManager_->addNetwork(std::move(M_), cctx);
     EXIT_ON_ERR(std::move(error));
+    // After partitioning, the original function may be removed. Need to update
+    // F_.
+    F_ = module->getFunctions().front();
```

Comment:
I'm curious, what do we use `F_` for after this point? It seems weird that we would just reset `F_` to the first function in the partition.

Reply:
We will use F_ to get the module and then get all the functions in this module for generating quantization info in Loader::generateAndSerializeQuantizationInfos. When the fucntion Loader::generateAndSerializeQuantizationInfos is called, the M_ has been removed already. So we need to use a function in M_ to reach all functions in M_.

Reply:
Got it -- but we already have `M_` in the Loader, so could we just do `getModule()`?

Reply:
getModule() here is { return F_->getParent();}  So we still need the updated F_  (the original F_ maybe erased from this module after partitioning) .  Now the code in  generateAndSerializeQuantizationInfos is
```
 for (auto F : getModule()->getFunctions()) {
    std::vector<NodeQuantizationInfo> tmp =
        quantization::generateNodeQuantizationInfos(bindings, F, loweredMap_,
                                                    quantizationSchema,
                                                    quantizationPrecision);
    QI.insert(QI.end(), tmp.begin(), tmp.end());
  }
```